istioctl/pkg/writer/pilot/testdata/multiStatusMultiPilot.txt

```diff
@@ -1,4 +1,4 @@
-NAME       CDS                            LDS          EDS                            RDS          PILOT      VERSION
-proxy1     STALE (1h0m0s)                 SYNCED       SYNCED                         NOT SENT     pilot1     1.1
-proxy2     STALE (1h0m0s)                 SYNCED       STALE (1h0m0s)                 SYNCED       pilot2     1.1
-proxy3     STALE (Never Acknowledged)     NOT SENT     STALE (Never Acknowledged)     SYNCED       pilot3     1.1
```

Comment:
It should still be possible to detect if it's never been acknowledged. I would think it's an empty UUID but check in the Pilot code.

Reply:
agree with @liamawhite . If we can detect the never acknowledged, that is better.

Reply:
yes, agree with @liamawhite . I just made a change to show the  'STALE (Never Acknowledged)'  when there is never acknowledged.

Reply:
commit can be found [here](https://github.com/istio/istio/pull/16477/commits/d670ae5d35923bed06ca2b6042ca18e0e61b58c1)